packages/editor/CodeMirror/CodeMirrorControl.test.ts

```diff
@@ -57,4 +57,23 @@ describe('CodeMirrorControl', () => {
 		expect(control.execCommand('myTestCommand')).toBe('test');
 		expect(command).toHaveBeenCalledTimes(1);
 	});
+
+	it('toggleComment should work', () => {
```

Comment:
Please use the same pattern as other tests - `it('should...`. And generally we should use plain English in the description so something `it('should toggle comments'...`